app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -43,6 +46,27 @@ public static void setImageDrawable(
     imageView.setImageResource(drawableResourceId);
   }
 
+  /**
+   * Allows binding drawables to an [ImageView] via "android:src".
```

Comment:
Should be declarative on the action being performed, e.g.: ``Binds a drawable indicated by {@link ChapterPlayState} to an {@link ImageView} via "android:src".``

(Note the ``@link`` syntax since this isn't a KDoc).

Reply:
Done. Thanks